tests/www/apply/test_process.py

```diff
@@ -1733,7 +1737,7 @@ def test_eligibility_for_siae_with_suspension_sanction(self, client):
 
         job_application = JobApplicationSentByPrescriberOrganizationFactory(
             state=job_applications_enums.JobApplicationState.PROCESSING,
-            job_seeker=JobSeekerFactory(with_address=True),
+            job_seeker=JobSeekerFactory(with_address=True, for_snapshot=True, born_in_france=True),
```

Comment:
Beaucoup de `for_snapshot=True` ont été ajoutés dans cette PR mais semblent inutiles. Mieux vaut n’utiliser le trait que lorsqu’on en a besoin.

```suggestion
            job_seeker=JobSeekerFactory(with_address=True, born_in_france=True),
```